### PR TITLE
Reduce message update size returned by websearch

### PR DESCRIPTION
**Pull request by nsarrazin**
The websearch was returning updates which contained a lot of information we don't use in the front-end. 

This PR removes the data from the DB and remove the unnecessary data from websearch updates.

### CODE REVIEW

**Comment by nsarrazin** [comment]
Since the front-end constructs the websearch widget using the message updates, i don't think we need to be storing the full websearch object in the DB, wdyt ?

File: src/lib/types/MessageUpdate.ts [hunk 1]
```diff
@@ -59,9 +59,8 @@ export interface MessageWebSearchSourcesUpdate
 	sources: WebSearchSource[];
 }
 export interface MessageWebSearchFinishedUpdate
-	extends BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Finished> {
-	webSearch: WebSearch;
-}
+	extends BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Finished> {}
```
Comment (mishig25) [inline src/lib/types/MessageUpdate.ts:62]:
```suggestion
export type MessageWebSearchFinishedUpdate = BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Finished>
```

**Review by mishig25 — APPROVED** [review]
lgtm !

File: src/lib/migrations/routines/06-trim-message-updates.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+export interface MessageWebSearchSourcesUpdate
+	extends BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Sources> {
+	message: string;
+	sources: WebSearchSource[];
+}
+export interface MessageWebSearchFinishedUpdate
+	extends BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Finished> {
+	webSearch: WebSearch;
+}
```
Comment (Saghen) [inline src/lib/migrations/routines/06-trim-message-updates.ts:24]:
```suggestion
interface MessageWebSearchSourcesUpdate
	extends BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Sources> {
	message: string;
	sources: WebSearchSource[];
}
interface MessageWebSearchFinishedUpdate
	extends BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Finished> {
	webSearch: WebSearch;
}
```

**Review by Saghen — APPROVED** [review]
lgtm! just one nit

File: src/lib/types/MessageUpdate.ts (hunk 1 above)
Comment (mishig25) [reply src/lib/types/MessageUpdate.ts]: handled in https://github.com/huggingface/chat-ui/pull/1179/commits/154d896bc4e56b44301cccb1375f70e4c525fb8f

**Pull request merged**